src/rootfstreemanager.cc

```diff
@@ -0,0 +1,15 @@
+#include "rootfstreemanager.h"
+
+bool RootfsTreeManager::fetchTarget(const Uptane::Target &target, Uptane::Fetcher &fetcher, const KeyManager &keys,
+                                    const FetcherProgressCb &progress_cb, const api::FlowControlToken *token) {
+  if (!target.IsOstree()) {
```

Comment:
not your bug, but it seems like best way to handle this would be to put such a check in ostreemanager::pull logic since its the method requiring this.  no point fixing here i guess.

Reply:
ok, just wanted to minimize changes to libaktualizr, but I think your're right, it's better to put into ostreemanager::pull although I feel like we will bypass it sooner or later since we have its all functional blocks in the scope of aklite project.

Reply:
updated.